apps/custom-trigger/src/services/monday-service.js

```diff
@@ -0,0 +1,45 @@
+const initMondayClient = require('monday-sdk-js');
+
+class MondayService {
+  static async getColumnValue(token, itemId, columnId) {
+    try {
+      const mondayClient = initMondayClient();
+      mondayClient.setToken(token);
+
+      const query = `query($itemId: [Int], $columnId: [String]) {
+        items (ids: $itemId) {
+          column_values(ids:$columnId) {
+            value
+          }
+        }
+      }`;
+      const variables = { columnId, itemId };
+
+      const response = await mondayClient.api(query, { variables });
+      return response.data.items[0].column_values[0].value;
```

Comment:
dangerous, need to check items.length and column_values.length, to not fail on the empty arrays